data/io.github.pragha_music_player.metainfo.xml

```diff
@@ -5,12 +5,30 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
   <name>Pragha Music Player</name>
+  <name xml:lang="es">Reproductor de música Pragha</name>
   <summary>Manage and listen to music</summary>
+  <summary xml:lang="bg">Слушайте и управлявайте музиката си</summary>
+  <summary xml:lang="ca_ES">Escolteu i gestioneu la música</summary>
+  <summary xml:lang="cs">Spravovat a poslouchat hudbu</summary>
+  <summary xml:lang="de">Musik verwalten und anhören</summary>
+  <summary xml:lang="es">Escuche y administre su música</summary>
+  <summary xml:lang="fr">Gérer et écouter de la musique</summary>
+  <summary xml:lang="it">Gestisci e ascolta la musica</summary>
+  <summary xml:lang="ko_KR">음악을 관리하고 들으세요</summary>
+  <summary xml:lang="lt">Tvarkyti ir klausytis muzikos</summary>
+  <summary xml:lang="nl">Beheer en luister naar muziek</summary>
+  <summary xml:lang="pt">Gestão e reprodução de músicas</summary>
+  <summary xml:lang="ru">Слушайте и управляйте музыкой</summary>
+  <summary xml:lang="tr">Müziği yönetin ve dinleyin</summary>
+  <summary xml:lang="uk">Слухайте та керуйте музикою</summary>
+  <summary xml:lang="zh_CN">管理和听音乐</summary>
   <description>
     <p>
-      Pragha is a Lightweight Music Player for GNU/Linux, based on Gtk, sqlite, and
-      completely written in C, constructed to be fast, light, and simultaneously tries
-      to be complete without obstructing the daily work.
+      Pragha is a Lightweight Music Player for GNU/Linux, based on Gtk3, sqlite,
+      and completely written in C, constructed to be fast, light, and
+      simultaneously tries to be complete without obstructing the daily work.
+      Although it was especifically designed for Xfce4 desktop, it integrates
```

Comment:
I liked this change, but I would say that it is 'thought' for the Xfce desktop, but not specially designed, due it does not fully follow its standards either. :sweat_smile: